src/grid/InstallmentPlanGridLegend.php

```
@@ -0,0 +1,82 @@
+
+declare(strict_types=1);
+
+namespace hipanel\modules\stock\grid;
+
+use hipanel\modules\stock\models\InstallmentPlan;
+use hipanel\widgets\gridLegend\BaseGridLegend;
+use hipanel\widgets\gridLegend\GridLegendInterface;
+use yii\helpers\Html;
+use Yii;
+
+class InstallmentPlanGridLegend extends BaseGridLegend implements GridLegendInterface
```

PHPStan (2.1.46)

[error] Class hipanel\modules\stock\grid\InstallmentPlanGridLegend extends unknown class hipanel\widgets\gridLegend\BaseGridLegend.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)